modules/Campaigns/ProcessBouncedEmails.php

```diff
@@ -166,15 +210,12 @@ function campaign_process_bounced_emails(&$email, &$email_header)
     $emailFromAddress = $email_header->fromaddress;
     $email_description = $email->raw_source;
 
-    //if raw_source is empty, try using the description instead
-    if (empty($email_description)) {
-        $email_description = $email->description;
-    }
-
     $email_description .= retrieveErrorReportAttachment($email);
 
+    // TODO: Delete attachments after this?
```

Comment:
Do not leave TODO comment in the code. It's implicate you miss something?

Reply:
It's just that I searched for related bug reports before I started working on this and this would be a place for fixing it, but I haven't looked into it. But that's another issue. I've removed the TODO